### DISCUSSION TITLE
Task Claiming, pls comment below.

### DISCUSSION THREAD

**Discussion by miketang84** [openra-rs/learn-openra#42] [General]
Pls claim one or more subsystems listed below by modifying this README file and do PR:

- game bootstrap logic(炸鸡, 隐, mengfanxing)
- graphic rendering-cross platform (including fonts)(saberuster, 是帽不是猫, num4, dunyi, birdinforest{weixin: Derek}, Kevin, fengqi)
- ecs subsystem (马许, minghuaw)
- player subsystem
- game UI widgets (Mike, mengfanxing)
- AI finding and others related (BluRadius, DawnMagnet, wushshsha)
- Network layer, logic (saberuster, cmoigo, nintha, daozhu，[LJason](https://github.com/LJason77), YuaXan)
- audio subsystem (BluRadius)
- map subsystem（是帽不是猫, birdinforst{weixin: Derek}, wzf）
- map editor(leanwy)
- hotkey subsystem
- mod subsystem (including lua script)(daozhu)
- testing (cmoigo)
- scripting subsystem
- task design and arragement subsystem (including lua script)
- file, format, parsing (JoJo)
- inputting, keyborad and mouse, cursor
- game directives
- fighting subsystem
- online fighting subsystem
- Actor design(num4, daozhu, minghuaw)
- model parsing and resource loading subsystem (是帽不是猫, 罪予)

Each item could be claimed by more than one persons :D

**Comment by mengFxing** [comment]
mengfanxing  game bootstrap logic game UI widgets

**Reply by miketang84** [reply]
updated upon.

**Comment by jm-observer** [comment]
fengqi:graphic rendering-cross platform

**Comment by wushshsha** [comment]
wushshsha: AI finding and others related

**Comment by minghuaw** [comment]
minghuaw (微信 NilNullNone): ECS subsystem, Actor design
